lib/svgo.js

```diff
@@ -20,7 +20,7 @@ var SVGO = module.exports = function(config) {
     this.config = CONFIG(config);
 };
 
-SVGO.prototype.optimize = function(svgstr) {
+SVGO.prototype.optimize = function(svgstr, info) {
```

Comment:
yea sorry again to be annoying, after that I stop:), but I think the best is to have an optional `prefix` property at the root of the config and pass it to any plugin that needs it like https://github.com/svg/svgo/blob/master/lib/svgo/config.js#L34, it's more consistent with current code. The other option would be to refactor more and do  `var svgo = require('svgo'); svgo.optimize(svgStr, options);` passing whole config at each call, possibly in the form `{common: {floatPrecision:.., prefix: '..'}, plugins: [...]}` so each plugin receive common and its plugin specific options. This plugin is interesting, even if it doesn't really minify, I use svgo to make png/jpg previews of them, and I do that in iframes to avoid those conflicts, so your way could make it work too.